code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -50,8 +68,21 @@
 
 			if(!registered_name)
 				registered_name = I.registered_name
-				desc = "Owned by [I.registered_name]."
+				desc += " Owned by [I.registered_name]."
+				to_chat(user, "<span class='notice'>You swipe your ID and take ownership of the [src].</span>")
 		else
 			to_chat(user, "<span class='danger'>Access Denied.</span>")
 	else
 		return ..()
+
+// This will let you wipe the owner data, for business or pleasure (or just to reset it for the next user, you sick bastard)
+/obj/structure/closet/secure_closet/personal/sleeper/attackby(obj/item/W, mob/user, params)

```

Comment:
make this also happen if the pod is emagged

Reply:
or use another emagging effect, idk

Reply:
keep in mind these are instanced from lockers- emagging should open and break them, which is functionally identical to removing the owner data